<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox_port">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect Arduino</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_connect">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Pair</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_boud">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>115200</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Boud</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_plot" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>991</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveTxt">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>480</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save data txt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_savePng">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>510</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save data png</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>631</width>
     <height>171</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Counter</string>
    </attribute>
    <widget class="QLCDNumber" name="CNT_lcdNumber_counts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Counts</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="CNT_spinBox_intTime">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>151</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Integration Time (ms)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>External Ref</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>HRTIM</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TIM 2</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>20</width>
       <height>111</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Internal Ref</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>TCSPC</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Avg counts</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="TCSPC_lcdNumber_counts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="TCSPC_widget_timeGate" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>91</width>
       <height>101</height>
      </rect>
     </property>
     <widget class="QRadioButton" name="TCSPC_radioButton_HRTIMprresc0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>2.08 ns</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="TCSPC_radioButton_HRTIMprresc1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>4.17 ns</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="TCSPC_radioButton_HRTIMprresc2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>8.33 ns</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Time gate</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="TCSPC_radioButton_HRTIMprresc3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>16.66 ns</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TCSPC_widget_chooseTimer" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QRadioButton" name="TCSPC_radioButton_selectHRTIM">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>HRTIM</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="TCSPC_radioButton_selectTIM2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TIM 2</string>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>21</width>
       <height>141</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Preescaler</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="TCSPC_spinBox_TIM2preesc">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="TCSPC_label_timeGate">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>151</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Time gate: 4.17 ns</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_trigger_cycles" native="true">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Trigger cycles</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="TCSPC_spinBox_triggerCycles">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_clearPlot">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Plot</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
